operator/v1/register.go

```diff
@@ -62,6 +62,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ServiceCatalogControllerManagerList{},
 		&IngressController{},
 		&IngressControllerList{},
+		&Metal3ProvisioningController{},
```

Comment:
Why is this called controller?
IngressController is the per object config which instantiates a new specific ingress controller pod which is not the case here.

I think this should be called just `Metal3Provisioning` or similar.

Reply:
It's config for the Metal3 provisioning controller, and related containers - Metal3ProvisioningConfig sounds fine to me, but we already went over some of this in the enhancement PR - @sadasu were there specific reasons for the current naming?

Reply:
@enxebre @shardy the naming was left partially unresolved in the enhancement proposal. I think Metal3Provisioning is a better name for us. Making the necessary changes.